### PR TITLE
Change the block editor inserter quick inserter border color

### LINKED ISSUE #27772: Quick Inserter: use inset border and rbga color value
The quick inserter has a standard border (outset) that is #ccc.

With the addition of the 'browse all' button in black at the bottom it creates an unfortunate fuzz between the black button, the gray border, and the black box shadow:

image

We should instead use an inset border like `border: 1px inset rgba(0,0,0,.05);` (or some other decimal value of black that's close to #ccc). The inset border overlaps the black button and the alpha property makes the border invisible on top of it.

Which looks like this:

Screen Shot 2020-12-16 at 5 44 26 PM

It's just a bit cleaner.

### PR DESCRIPTION
**Pull request by hsingyuc**
## Description
Fix #27772 
Change the block editor inserter quick inserter border color so the inserter button does not have a gray border.

## How has this been tested?
Click on the add block button on both post and page, and check if it renders as expected

## Screenshots 
1. Before
before

2. After
after

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-editor/src/components/inserter/style.scss [hunk 1]
```diff
@@ -16,6 +16,29 @@ $block-inserter-tabs-height: 44px;
 	}
 }
 
+.block-editor-inserter__popover.is-quick {
+	.components-popover__content {
+		border: none;
+
+		.block-editor-inserter__quick-inserter > * {
```
Comment (jasmussen) [inline packages/block-editor/src/components/inserter/style.scss:23]: The use of wildcards and first/last-child selectors here feels a little hard to parse. I think I understand what it does, and that this is all fine, but are there some explicit CSS classes we could've used instead?

**Review by jasmussen — APPROVED** [review]
Thank you for the PR. It definitely works as intended. Before:

Screenshot 2021-01-04 at 11 40 54

Screenshot 2021-01-04 at 11 48 57

After:

Screenshot 2021-01-04 at 11 46 25

Screenshot 2021-01-04 at 11 47 45

It also avoids using transparencies. Transparencies aren't a problem on their own, but it's nice that we can use the preset grays defined in the color variable sheet. Nice.

I left some comments on the use of wildcards and first/last childs. But because it works as it should in my testing, this seems fine to me, if there aren't any good alternatives.

Thanks!

File: packages/block-editor/src/components/inserter/style.scss (hunk 1 above)
Comment (hsingyuc) [reply packages/block-editor/src/components/inserter/style.scss:23]: @jasmussen  Thank you for the review! I've thought about using explicit classes, but we can’t predict which type of items or how many will be visible. This seems the most future proof way. Because those components can be visible or hidden, and it’s possible someone may add in a new div later?

Comment (jasmussen) [reply packages/block-editor/src/components/inserter/style.scss:23]: I figured it might be something like that. But we can try this for now and see if any problems pop up, it seems to work in my tests. I'll restart the e2e tests and see if we can get them to pass, then we can land this. Thanks again.
